fundamental-concepts/contract-keys/1_chasing_contract_ids.md

```diff
@@ -48,7 +48,7 @@ curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $A
     \"username\": \"$ALICE\",
     \"following\": []
   }}" localhost:7575/v1/create | tee result
-```{{execute T1}}
+```{{execute T2}}
```

Comment:
instead of changing all of this, why don’t you add the installaiton of the vscode extension which includes installation of the 2.0 SDK like we do for all other scenarios?

Reply:
This won't execute in T1 as T1 is running the sandbox and the JSON API from the `create-daml-app` folder. Running the commands in the second terminal looks cleaner. Doesn't have to do much with SDK version and the extension

Reply:
well don’t you want to fix it to use 2.0 though? or are you just relying on it working after the base image is updated.

Reply:
Yep, relying on the changes with that are already in the `environments/daml-sdk-env/build` folder. 

- `2_installsdk.sh` has already `export SDK_VERSION=2.0.0` in it
- `3_configsdk.sh` has 
```
cd /tmp
curl -sSL https://gist.githubusercontent.com/cocreature/b616b7ff1244b227ee17cdc476695435/raw/fd337aab4460e2e81074bc3dfd3e527b52c233d5/patch-extension.sh > patch-extension.sh
chmod +x patch-extension.sh
./patch-extension.sh $(find ~/.daml/sdk/2.*/studio/daml-bundled.vsix) patched-daml-bundled.vsix
unzip patched-daml-bundled.vsix -d daml-bundled
mv daml-bundled/extension /opt/.katacodacode/extensions/daml-bundled
rm -Rf /tmp/daml-bundled
rm -Rf daml-bundled
```

Once the 2.0 is out I'll need to clean up the init scripts where we do all this work now - this is a tmp fix so we don't have to type the commands all the time